source/server/main.qc

```diff
@@ -377,6 +377,7 @@ void() worldspawn =
 	clientstat(STAT_VIEWZOOM, EV_FLOAT, viewzoom);
 	clientstat(STAT_MAXHEALTH, EV_FLOAT, max_health);
 	clientstat(STAT_PERKS, EV_FLOAT, perks);
+	clientstat(STAT_VELOCITY, EV_VECTOR, velocity);
```

Comment:
We already write the client's velocity when we send the player to CSQC for drawing and physics, can we reuse that instead of writing it again?

Reply:
addressed in [81b0333](https://github.com/nzp-team/quakec/pull/95/commits/81b033318d9613e7c5238828ec00cae097fe5846)